pkg/kubelet/certificate/bootstrap/bootstrap.go

```diff
@@ -26,12 +29,13 @@ import (
 
 	"k8s.io/klog"
 
+	certificates "k8s.io/api/certificates/v1beta1"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/types"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes/scheme"
-	certificates "k8s.io/client-go/kubernetes/typed/certificates/v1beta1"
+	certificatesclient "k8s.io/client-go/kubernetes/typed/certificates/v1beta1"
```

Comment:
`certificatesv1beta1` would be consistent with what we do elsewhere.

Reply:
Done